src/lib/pure/ketrew_edsl.mli

```diff
@@ -251,3 +252,56 @@ val pbs :
   [> `Long_running of string * string ]
 (** Create a “PSB” build process. *)
 
+
+val yarn_application :
+  ?host:Ketrew_host.t ->
+  ?daemonize_using:[ `Nohup_setsid | `Python_daemon ] ->
+  ?daemon_start_timeout:float ->
+  Ketrew_program.t -> [> `Long_running of string * string ]
+(** Create a build process that requests resources from Yarn, the
+    command must be an application in the Yarn sense (i.e.
+    a program that is going to contact Yarn by itself to request
+    containers):
```

Comment:
"build process" is part of  Ketrew's own annoying (but unavoidable) jargon: http://seb.mondet.org/software/ketrew/Glossary.html

We can change it but that should a different issue (global change alll over place).